src/Entity/ProductTrait.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Locastic\SyliusCatalogPromotionPlugin\Entity;
+
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\Model\ChannelPricingInterface;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+
+trait ProductTrait
```

Comment:
@dujesaric maybe to rename it to ProductCatalogTrait - just to be clear what it is?